hack/make/test-integration-cli

```diff
@@ -2,7 +2,7 @@
 set -e
 
 bundle_test_integration_cli() {
-	TESTFLAGS="$TESTFLAGS -check.v"
+	TESTFLAGS="$TESTFLAGS -check.v -check.timeout=${TIMEOUT} -timeout=360m"
```

Comment:
yeah, none of the built-in flags need the `test` prefix (or at least none that I've used).